lib/mongo/server.rb

```diff
@@ -61,6 +61,9 @@ def initialize(address, cluster, monitoring, event_listeners, options = {})
       monitor = options.delete(:monitor)
       @options = options.freeze
       @event_listeners = event_listeners
+      @connection_id_gen = Class.new do
+        include Id
+      end
```

Comment:
Let's not create class definitions when unnecessary and instead define a class in the conventional way, and instantiate it here.

Reply:
OK.